packages/create-plugin/scripts/generate-datasource-without-backend.js

```diff
@@ -0,0 +1,24 @@
+const nodePlop = require('node-plop');
+const path = require('path');
+const fs = require('fs');
+
+async function generatePanel() {
```

Comment:
**Nit:** probably `generateDatasource()` instead, although `generate()` would be probably better as it's "copy-paste safer" :)